specification/columns/commitmentdiscountpurchasedquantity.md

```diff
@@ -0,0 +1,38 @@
+# Commitment Discount Purchased Quantity
+
+Commitment Discount Purchased Quantity is the total, positive quantity of [Commitment Discount Units](#commitmentdiscountunit) that is eligible for consumption from a [*commitment discount's*](#glossary:commitment-discount) purchase over a [*row's*](glossary:row) [*charge period*](#glossary:chargeperiod). The CommitmentDiscountPurchaseQuantity column is only applicable to *commitment discounts* and not [*negotiated discounts*](#glossary:negotiated-discount).
```

Comment:
@udam-f2 @kk09v @flanakin There are conflicting viewpoints around whether to use `amortized` or not for purchases.  Can you comment to drive towards a solution?

Reply:
I haven't internalized the scenario where this will be used enough to know whether it should be amortized or not. Being amortized gives insight into the amortization logic that cannot be pulled from anywhere else. The purchase quantity is already in the data (albeit up to 5 years before). Depends on what we're prioritizing.

Reply:
@flanakin thoughts on this:
https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/400#discussion_r1732252309

Reply:
Going with Udam/Karl's suggestion to use "consumed"/"consumption"